### PR TITLE
Update How to generate Google API keys.md

### PR DESCRIPTION
**Pull request by rafek1241**
Because your documentation is invalid right now because of new OAuth.

### CODE REVIEW

**Comment by fregante** [comment]
Thank you for the PR. Does that work? It nearly the same:

```
https://accounts.google.com/o/oauth2/auth?response_type=code&scope=https://www.googleapis.com/auth/chromewebstore&client_id=$CLIENT_ID&redirect_uri=urn:ietf:wg:oauth:2.0:oob
```


to step 15 https://github.com/fregante/chrome-webstore-upload/blob/main/How%20to%20generate%20Google%20API%20keys.md

```
https://accounts.google.com/o/oauth2/auth?response_type=code&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fchromewebstore&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&access_type=offline&approval_prompt=force&client_id=YOUR_CLIENT_ID_HERE
```

If the URL suggested there works, it's best we update the guide here instead of pointing elsewhere

**Comment by rafek1241** [comment]
@fregante I wanted to point to official documentation because if it is near the creators of API, there is bigger chance that it's up to date. You won't need to track incoming changes and update this document.

**Comment by rafek1241** [comment]
Okay, i'm not sure if it works because for me that authorization is not working at all.
image
I've got such issue.

Request that I used:
```
https://accounts.google.com/o/oauth2/auth?response_type=code&scope=https://www.googleapis.com/auth/chromewebstore&client_id=XXXX.apps.googleusercontent.com&redirect_uri=urn:ietf:wg:oauth:2.0:oob
```

**Comment by fregante** [comment]
Yeah that's the point. I don't know if that's a new document and it can replace the current one. When I wrote it there was nothing online if not several generic OAuth pages that are written for applications (for other users) rather than self-serving fake apps. 

I don't have time to investigate right now but I hope someone can figure it out.

File: How to generate Google API keys.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # How to generate Google API keys
 
+> Disclaimer: The most recent information can be found at [chrome developers documentation](https://developer.chrome.com/docs/webstore/using_webstore_api/)
```
Comment (fregante) [inline How to generate Google API keys.md:3]: If you change this "disclaimer" to say something like "Step 15 is out of date. Please refer to issue XYZ or look for an alternative in DOCS_LINK. PR welcome to fix it"

**Comment by rafek1241** [comment]
For everyone who have the same problem - there is solution to mitigate this problem (kudos for @louisgv):

https://github.com/fregante/chrome-webstore-upload/issues/59#issuecomment-1096067293
> Just released [gcp-refresh-token](https://www.npmjs.com/package/gcp-refresh-token), source code [here](https://github.com/plasmo-corp/gcp-refresh-token).
> 
> Let me know if you have any issue (preferably creating an issue under `gcp-refresh-token`). To use, download your OAuth Client json key into a `key.json` file, then run:
> 
> ```
> npx gcp-refresh-token
> 
> # or
> 
> pnpm dlx gcp-refresh-token
> ```
> 
> Refers to the [readme](https://github.com/plasmo-corp/gcp-refresh-token) for more info!
> 
> --[/U64eCftcNBXxHZRu3](https://www.plasmo.com/engineering/log/2022.04)

**Pull request closed without merging**